salt/cloud/__init__.py

```diff
@@ -874,7 +874,7 @@ def image_list(self, lookup="all"):
                 )
         return data
 
-    def size_list(self, lookup="all"):
+    def size_list(self, lookup="all", **kwargs):
```

Comment:
I prefer we do not accept kwargs unless there is a very compelling reason. Can this be changed to accept the `location` argument instead of `**kwargs`